src/VCDiff/Decoders/WindowDecoder.cs

```diff
@@ -6,7 +6,13 @@ namespace VCDiff.Decoders
 {
     internal class WindowDecoder
     {
-        private const uint HardMaxWindowSize = 1u << 24;
+
+        /**
+         * The default maximum target file size (and target window size) 
+         */
+        public const int DefaultMaxTargetFileSize = 67108864;  // 64 MB
```

Comment:
```suggestion
        private static readonly int DefaultMaxTargetFileSize = 67108864;  // 64 MiB
```

Reply:
This value must be a `const` to be used in as default value in the constructor.  
And it must be public (or internal) to be used in `VcDecoder`.

So if I apply this, the build will fail.

Reply:
Ok, sounds good.